### PR TITLE
Implemented mongodb connection in data.js and updated package.json

### PR DESCRIPTION
**Pull request by develra**

### CODE REVIEW

File: lib/diff/data.js [hunk 1]
```diff
@@ -18,13 +19,20 @@ var plugin = function(diff) {
...
-        
-        // TODO: return the server object 
-        // callback(null, mongodb)
-        
-        callback('$mongodb has not been implemeneted')
-    })
+        var mongoClient = require('mongodb').MongoClient;
+	mongoClient.connect("mongodb://localhost:27017/fcq", function (err, db) {
+	    if(!err){
+		debug("Connected to fcq database")
```
Comment (ianks) [inline lib/diff/data.js]: Consistent line-spacing, plz :cat:

Comment (ianks) [inline lib/diff/data.js]: There are a few spots where theindentation/styling is weird. IIRC you use VIM... there's a quick way to auto-indent a section:
1. Select the section you want to indent in visual mode (Ctrl-V)
2. Press `=`
3. Voila!

**Comment by ianks** [comment]
Looks good apart from my style nit-picks.

One last thing would be to keep your commit messages in the imperative tense for future commits. Here is a very worthwhile read which makes dealing with Git commits (and subsequently Git history) much easier: http://tbaggery.com/2008/04/19/a-note-about-git-commit-messages.html

**Comment by develra** [comment]
Oops, sorry about that. I was messing around with a config file and messed up my indenting. Thanks for the visual mode trick! http://www.emacswiki.org/emacs/TabsSpacesBoth

Git stuff makes sense. I'll be sure to use shorter commits in the future.

**Comment by ianks** [comment]
Your commit message length and content was good. Just the tense. Minor thing but good to know since it has become the standard in most projects.

**Pull request merged**